core/src/scene/sceneLoader.cpp

```diff
@@ -177,470 +141,323 @@ void SceneLoader::applyGlobals(Node root, Scene& scene) {
     }
 }
 
-bool SceneLoader::applyConfig(Platform& _platform, const std::shared_ptr<Scene>& _scene) {
-
-    Node& config = _scene->config();
-
-    // Instantiate built-in styles
-    _scene->styles().emplace_back(new PolygonStyle("polygons"));
-    _scene->styles().emplace_back(new PolylineStyle("lines"));
-    _scene->styles().emplace_back(new DebugTextStyle("debugtext", _scene->fontContext(), true));
-    _scene->styles().emplace_back(new TextStyle("text", _scene->fontContext(), true));
-    _scene->styles().emplace_back(new DebugStyle("debug"));
-    _scene->styles().emplace_back(new PointStyle("points", _scene->fontContext()));
-    _scene->styles().emplace_back(new RasterStyle("raster"));
-
-    if (config["global"]) {
-        applyGlobals(config, *_scene);
+void SceneLoader::applyScene(const Node& _node, Color& backgroundColor,
+                             Stops& backgroundStops, Scene::animate& animated) {
+    if (!_node) { return; }
+    if (!_node.IsMap()) {
+        LOGNode("Invalid 'scene' section", _node);
+        return;
     }
 
+    if (const Node& background = _node["background"]) {
+        if (const Node& colorNode = background["color"]) {
+            if (!StyleParam::parseColor(colorNode, backgroundColor)) {
 
-    if (Node sources = config["sources"]) {
-        for (const auto& source : sources) {
-            std::string srcName = source.first.Scalar();
-            try { loadSource(_platform, srcName, source.second, sources, _scene); }
-            catch (YAML::RepresentationException e) {
-                LOGNode("Parsing sources: '%s'", source, e.what());
+                Stops stopsResult = Stops::Colors(colorNode);
+                if (stopsResult.frames.size() > 0) {
```

Comment:
nit: prefer `.empty()`

Reply:
Sure. Left-over from old state. Should these go in this PR?